src/core/python/distr_1d_cond_v.cpp

```diff
@@ -0,0 +1,114 @@
+#include <drjit/dynamic.h>
+#include <drjit/tensor.h>
+#include <mitsuba/core/distr_1d_cond.h>
+#include <mitsuba/core/spectrum.h>
+#include <mitsuba/python/python.h>
+#include <nanobind/stl/pair.h>
+#include <nanobind/stl/string.h>
+#include <nanobind/stl/tuple.h>
+#include <nanobind/stl/vector.h>
+
+MI_PY_EXPORT(ConditionalIrregular1D) {
+    MI_PY_IMPORT_TYPES()
+
+    using TYPE                   = UnpolarizedSpectrum;
+    using ConditionalIrregular1D = mitsuba::ConditionalIrregular1D<TYPE>;
+    using FloatStorage           = DynamicBuffer<Float>;
+
+    MI_PY_CHECK_ALIAS(ConditionalIrregular1D, "ConditionalIrregular1D") {
+        nb::class_<ConditionalIrregular1D>(m, "ConditionalIrregular1D")
+            .def(nb::init<>(), D(ConditionalIrregular1D))
+            .def(nb::init<const FloatStorage &, const FloatStorage &,
+                          const std::vector<FloatStorage> &, bool, bool>(),
+                 "nodes"_a, "pdf"_a, "nodes_cond"_a, "enable_sampling"_a = true,
+                 "nearest"_a = false, D(ConditionalIrregular1D, ConditionalIrregular1D, 2))
+            .def(
+                "eval_pdf",
+                [](const ConditionalIrregular1D *distr, TYPE x,
+                   std::vector<TYPE> cond, Mask active = true) {
+                    return distr->eval_pdf(x, cond, active);
+                },
+                "x"_a, "cond"_a, "active"_a = true, D(ConditionalIrregular1D, eval_pdf))
+            .def(
+                "eval_pdf_normalized",
+                [](const ConditionalIrregular1D *distr, TYPE x,
+                   std::vector<TYPE> cond, Mask active = true) {
+                    return distr->eval_pdf_normalized(x, cond, active);
+                },
+                "x"_a, "cond"_a, "active"_a = true, D(ConditionalIrregular1D, eval_pdf_normalized))
+            .def(
+                "sample_pdf",
+                [](const ConditionalIrregular1D *distr, TYPE u,
+                   std::vector<TYPE> cond, Mask active = true) {
+                    return distr->sample_pdf(u, cond, active);
+                },
+                "u"_a, "cond"_a, "active"_a = true, D(ConditionalIrregular1D, sample_pdf))
+            .def(
+                "integral",
+                [](const ConditionalIrregular1D *distr,
+                   std::vector<TYPE> cond) { return distr->integral(cond); },
+                "cond"_a, D(ConditionalIrregular1D, integral))
+            .def("tensor", &ConditionalIrregular1D::tensor,
+                 nb::rv_policy::reference_internal, D(ConditionalIrregular1D, tensor))
```

Comment:
Let's make this a read/write Python property. I don't think it's currently doing what you'd expect it to do. Please add a test for this too.